src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function getOrder($size, $gap, $index) {
+  @return ($size + $gap) * ($index - 1);
```

Comment:
Instead of this you could just set values starting from 0 instead of 1 (like indexes in arrays)